File: Pertemuan 6/tugassementara.py
print("Selamat Datang di Siakad UNY!")
print(".: Program Siakad :.")
print("1. Tambah Data ")
print("2. Edit Data")
print("3. Hapus Data ")
print("4. Lihat Data")
menuI = input("Menu yang ada pilih: ")
menuI = int(menuI)
if (menuI == 1) :
    print("----- Tambah Data -----")
    print("1. Tambah Data Mahasiswa")
    print("2. Tambah Data Mata Kuliah")
    print("3. Kembali")
    menuI2 = input("Menu yang ada pilih: ")
    menuI2 = int (menuI2)
    if (menuI2 == 1):
        namaMahasiswa = input("Masukkan nama mahasiswa: ")
        print("Mahasiswa yang Anda tambah adalah " + namaMahasiswa)
    elif (menuI2 ==2):
        mataKuliah = input("Masukkan mata kuliah: ")
        print("Mata kuliah yang Anda tambah adalah " + mataKuliah)
    elif (menuI2 == 3):
        print("Terima kasih")
    else:
        print("Menu tidak ada")
elif (int(menuI) >= 2 and int(menuI)<= 4):
    print("Fitur belum ada")
else:
    print("Menu belum tersedia")

File: Pertemuan 6/TugasSementara.java
import java.util.Scanner;

public class TugasSementara {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Selamat Datang di Siakad UNY!");
        System.out.println(".: Program Siakad :.");
        System.out.println("1. Tambah Data ");
        System.out.println("2. Edit Data");
        System.out.println("3. Hapus Data ");
        System.out.println("4. Lihat Data");

        System.out.println("Menu yang Anda pilih =");
        int menu1 = scanner.nextInt();
        if (menu1 == 1) {
            System.out.println();
            System.out.println("----- Tambah Data -----");
            System.out.println();

            System.out.println("1. Tambah Data Mahasiswa");
            System.out.println("2. Tambah Data Mata Kuliah");
            System.out.println("3. Kembali");
            System.out.println();

            System.out.print("Masukkan nama Mahasiswa :");
            String namaMahasiswa = scanner.nextLine();
            System.out.println("Mahasiswa yang Anda tambah adalah " + namaMahasiswa);

        } else if (menu1 == 2) {
            System.out.println();
            System.out.println("2. Tambah Data Mata Kuliah");
            System.out.println();

            System.out.print("Masukkan nama mata kuliah :");
            String namaMataKuliah = scanner.nextLine();
            System.out.println("Mata kuliah yang Anda tambah adalah " + namaMataKuliah);

        } else if (menu1 == 3) {
            System.out.println();
            System.out.println("------- Terimakasih -------");
        } else {
            System.out.println("Menu tidak ada");
        }

    }
}